apps/api/src/mail/mail.service.ts

```diff
@@ -32,7 +32,7 @@ export class MailService {
     if (!chesHost) return;
 
     const emailBody = {
-      from: mailOptions.from,
+      from: 'EHPRDoNotReply@gov.bc.ca',
```

Comment:
CHES seems to reject requests if the `from` is not `EHPRDoNotReply@gov.bc.ca`

Reply:
I have mixed feelings about hard coding this. Could we instead just make sure that we're passing the correct email address where this function is called? 
Also shouldn't this be EHPRDoNotReply@ehpr.gov.bc.ca?

Reply:
It is just for CHES temporarily. One for SES is set by terraform.